app/components/modals/speakerimg-cropper-modal.js

```diff
@@ -0,0 +1,42 @@
+import ModalBase from 'open-event-frontend/components/modals/modal-base';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Definition for rule 'ember-suave/lines-between-object-properties' was not found.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4844788)